build/dnvm.ps1

```diff
@@ -84,7 +84,7 @@ Set-Variable -Option Constant "CommandFriendlyName" ".NET Version Manager"
 Set-Variable -Option Constant "DefaultUserDirectoryName" ".dnx"
 Set-Variable -Option Constant "OldUserDirectoryNames" @(".kre", ".k")
 Set-Variable -Option Constant "RuntimePackageName" "dnx"
-Set-Variable -Option Constant "DefaultFeed" "https://www.nuget.org/api/v2"
+Set-Variable -Option Constant "DefaultFeed" "https://www.myget.org/F/aspnetvnext/api/v2"
```

Comment:
Ok. We'll have to account for this as part of our build.cmd (to have it pass the right `DNX_FEED` \ flag) in release \ master branches. https://github.com/aspnet/Universe/issues/215 to track this